plans/subplans/provision.pp

```diff
@@ -111,6 +129,7 @@
 
   if $provider == 'aws' {
     $_instance_image = $instance_image
+    $_windows_instance_image = $instance_image
```

Comment:
IMO, just fail the run if the windows parameters are set and the provider isn't Azure.

Reply:
Also. Sending undef later to the tfvars template will work beter